src/utils.rs

```diff
@@ -113,7 +116,7 @@ impl Decompose for RgbImage {
     }
 }
 
-pub fn merge_similarity_channels(input: &[&GraySimilarityImage; 3]) -> RGBSimilarityImage {
+pub(crate) fn merge_similarity_channels(input: &[&GraySimilarityImage; 3]) -> RGBSimilarityImage {
```

Comment:
thanks for the cleanup :)

Reply:
Of course, exposing utils as public added irrelevant things so it seemed like a good thing to fix :)